services/yaml/__init__.py

```diff
@@ -127,6 +131,22 @@ def save_repo_yaml_to_database_if_needed(current_commit, new_yaml):
             yaml_branch = read_yaml_field(new_yaml, ("codecov", "branch"))
             if yaml_branch:
                 repository.branch = yaml_branch
+
+            maybe_update_repo_bot(new_yaml, repository)
             repository.yaml = new_yaml
             return True
+
     return False
+
+
+def maybe_update_repo_bot(
```

Comment:
Oh thanks for this! I had done it [in API](https://github.com/codecov/codecov-api/blob/fcee05d457747c372dafa8dae3ef886da15d7eb4/codecov_auth/commands/owner/interactors/set_yaml_on_owner.py#L55) but I missed this being a thing in worker, so thanks!

Also, technically the trigger you pointed to no longer exists since we actually use the migrations that come from shared https://github.com/codecov/codecov-api/blob/fcee05d457747c372dafa8dae3ef886da15d7eb4/codecov/settings_base.py#L436 which I had deleted. We can technically get rid of the migrations folders for the apps in that settings file so it's less confusing. We haven't cause it was a "next step" in that whole model migration thing and never got to it :/